css/gallery.css

```diff
@@ -111,15 +150,13 @@
     -ms-transition: all 0.9s ease;
     /* Opera */
```

Comment:
What is Opera and why is in class properties?

Reply:
browser opera.